src/Microsoft.AspNet.Mvc.Razor/Compilation/CompilationFailure.cs

```diff
@@ -2,51 +2,40 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System.Collections.Generic;
-using Microsoft.AspNet.Diagnostics;
+using Microsoft.Framework.Runtime;
 
 namespace Microsoft.AspNet.Mvc.Razor
 {
     /// <summary>
-    /// Default implementation of <see cref="ICompilationFailure"/>.
+    /// <see cref="ICompilationFailure"/> for Razor parse failures.
     /// </summary>
-    public class CompilationFailure : ICompilationFailure
+    public class RazorCompilationFailure : ICompilationFailure
```

Comment:
I could go either way on this naming. There isn't really any functionality in this class that makes this "razor" specific.